content/lua/concepts/arrays/arrays.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'Arrays' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Arrays are used to store ordered collections of data.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
+  - 'Lua'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Tables'
+  - 'Arrays'
+  - 'Iterators'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'lua/arrays'
+---
+
+# Arrays
+
+**Arrays** are tables of ordered, stored **data**. They can hold any type of data, and can be cleared with `nil` values.
+
+
+## Syntax
+
+In Lua, Arrays can be created with curly braces `{}`. When storing values, arrays will automatically order values starting from `0` unless specified otherwise. For instance, the following code will store 100 values using an `iteration`.
+
+```lua
+
+a = {};
+
+for i = 1, 100 do
+    a[i] = 0
+end
+
+```
+
+If we print our array, we see that it now holds 100 values starting from 1. However, if we try printing any value out of this range, we will receive **nil** instead. This is because **nil** refers to data that is not there, which in our case would be true because we have only stored 100 values.
```

Comment:
I also think we can try to minimize the amount of "we" and "you" we use in these entries. These are more similar to our instructional content in courses.

Reply:
For example, 
`If we print our array, we see that it now holds 100 values starting from 1`
can become
`When the array is printed, it would display 100 values starting from 1`

Reply:
What I meant to say was that our index starts at 1. Also, I'll apply these changes to minimize the usage of "we" and "you".